### PR TITLE
Implement the Get function for aws-backend.

### PR DESCRIPTION
**Pull request by HanFa**

### CODE REVIEW

File: storage/store.go [hunk 1]
```diff
@@ -118,7 +120,27 @@ func (a *awsBackend) WatchList(ctx context.Context, key string, opts storage.Lis
...
+		VersionId: latestCommitted.VersionId,
+	})
+	if err != nil {
+		return err
+	}
+	raw, err := ioutil.ReadAll(data.Body)
+	if err != nil {
+		return err
+	}
+	return decode(a.codec, a.Versioner(), raw, objPtr, 1)
```
Comment (hanlins) [inline storage/store.go]: revision should not be hardcoded as `1`. Need to retrieve if from tags.

File: storage/txn.go [hunk 2]
```diff
@@ -193,7 +193,12 @@ func (t *txn) Update(ctx context.Context, key string, value []byte, origRev uint
 	panic("not implemented") // TODO: Implement
 }
 
-var _ Txn = &txn{}
```
Comment (hanlins) [inline storage/txn.go]: IMO no need for change unless you want to separate the transaction code into a separate package from the storage package. The original line is the typical way just for checking if the `txn` type has implemented `Txn` interface during compiling time. Let's keep it what it was.

**Comment by hanlins** [comment]
Overall looks good. It would be nice if you limit your changes only in `storage/store.go` and `storage/store_test.go`, so I it's not conflicting with some of my changes, thanks.

**Review by hanlins — CHANGES_REQUESTED** [review]
Just change `storage/store.go` and `storage/store_test.go`, and fix the revision number during decoding.

File: storage/store.go (hunk 1 above)
Comment (HanFa) [reply storage/store.go]: changed to get from tagset

File: storage/txn.go (hunk 2 above)
Comment (HanFa) [reply storage/txn.go]: sure, reverted

**Comment by HanFa** [comment]
thanks for reviewing @hanlins could you PTAL? I have changed the PR according to your comments.

**Review by hanlins — APPROVED** [review]

**Pull request merged**